index.html

```diff
@@ -47,6 +48,157 @@ <h1 class="introduction-title">Hey there, I’m Matama <span class="title-green"
     </ul>
   </section>
 
+<!-- works section -->
```

Comment:
:orange_circle: [OPTIONAL] I think it will be better if you wrap everything that belongs to the work section in a parent section container as follow :arrow_down: 
```
<section id="works-section">
    paste work section content here
</section> 
```